pkg/virt-operator/resource/generate/install/strategy.go

```diff
@@ -437,8 +479,29 @@ func mostRecentConfigMap(configMaps []*corev1.ConfigMap) *corev1.ConfigMap {
 	return configMap
 }
 
+func isEncoded(configMap *corev1.ConfigMap) bool {
+	_, ok := configMap.Annotations[v1.InstallStrategyConfigMapEncoding]
+	return ok
+}
+
+func getManifests(configMap *corev1.ConfigMap) (string, error) {
+	manifests, ok := configMap.Data["manifests"]
+	if !ok {
+		return "", fmt.Errorf("install strategy configmap %s does not contain 'manifests' key", configMap.Name)
+	}
+
+	if isEncoded(configMap) {
+		var err error
+
+		manifests, err = DecodeManifests([]byte(manifests))
```

Comment:
It's deliberate, otherwise `manifests` will be aliased by the inner scope declaration, in this way I keep using the same var. I know this is not great...